autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -352,16 +374,19 @@ def train_single_full(self, X_train, y_train, X_test, y_test, model: AbstractMod
             # hpo_models (dict): keys = model_names, values = model_paths
             try:  # TODO: Make exception handling more robust? Return successful HPO models?
                 if type(model) in [BaggedEnsembleModel, StackerEnsembleModel, WeightedEnsembleModel]:
-                    hpo_models, hpo_model_performances, hpo_results = model.hyperparameter_tune(X=X_train, y=y_train, k_fold=kfolds, scheduler_options=(self.scheduler_func, self.scheduler_options), verbosity=self.verbosity)
+                    hpo_models, hpo_model_performances, hpo_results = model.hyperparameter_tune(
+                        X=X_train, y=y_train, k_fold=kfolds, scheduler_options=(self.scheduler_func, self.scheduler_options), verbosity=self.verbosity)
                 else:
                     if (X_test is None) or (y_test is None):
-                        X_train, X_test, y_train, y_test = self.generate_train_test_split(X_train, y_train, test_size=0.2)  # TODO: Adjust test_size, perhaps user specified?
+                        X_train, X_test, y_train, y_test = self.generate_train_test_split(
+                            X_train, y_train, test_size=0.2)  # TODO: Adjust test_size, perhaps user specified?
                     hpo_models, hpo_model_performances, hpo_results = model.hyperparameter_tune(X_train=X_train, X_test=X_test,
```

Comment:
This is an example where a func returns multiple outputs which could make the params take up many lines